### DISCUSSION TITLE
A way to manage language server plugins

### DISCUSSION THREAD

**Discussion by LukaK** [mason-org/mason.nvim#180] [Q&A]
Hi,
just wondering, is there currently a better way to handle language server plugins then manually installing packages to their virtual environments?

So for example, for python language server, to enable mypy plugin you have to go to where python-laugnage-server is installed, activate virtualenvironment and install pyls-mypy package.

**Comment by williamboman** [comment]
Hello! For `pylsp` especially, there is actually a dedicated command to install extensions: [`:PylspInstall`](https://github.com/williamboman/mason-lspconfig.nvim/blob/main/lua/mason-lspconfig/server_configurations/pylsp/README.md). It's however currently only available if you use the `mason-lspconfig` plugin - I need to rethink it a bit.
